code/game/objects/structures/dropship_equipment.dm

```diff
@@ -306,7 +306,116 @@
 		linked_console.selected_equipment = src
 		to_chat(user, span_notice("You select [src]."))
 
+//////////////////////////////////// rappel system //////////////////////////////////////
+/obj/structure/dropship_equipment/rappel_system
+	equip_category = DROPSHIP_CREW_WEAPON
+	name = "rappel deployment system"
+	desc = "A system that deploys rappel rope to go down fast without the need for the tadpole to land, you need to designate the rappel point at the navigation computer."
+	dropship_equipment_flags = IS_INTERACTABLE|IS_NOT_REMOVABLE
+	icon_state = "rappel_hatch_closed"
+	point_cost = 0
+	resistance_flags = RESIST_ALL
+	anchored = TRUE
+	density = FALSE
+	var/area/A
+	var/turf/T
+	var/obj/effect/rappel_rope/rope
```

Comment:
No autodoc
Extremely bad var names

Correction: These vars should not need to exist at all and be passed within the procs instead when they are needed